Update installation-procedure.md (#2382)
cyberbotics/webots: docs/guide/installation-procedure.md

The export line should however be included in a configuration script like "/etc/profile", so that it is set properly for every session.

> > **Note**: You will need to install *make* and *g++* to compile your own robot controllers. Other particular libraries could also be required to recompile some of the distributed binary files. The package names could slightly change on different releases and distributions. In this case an error message will be printed in the Webots console mentioning the missing dependency.

Reviewer: We try to write only one sentence per line:
```suggestion
> **Note**: You will need to install *make* and *g++* to compile your own robot controllers.
Other particular libraries could also be required to recompile some of the distributed binary files.
The package names could slightly change on different releases and distributions.
In this case an error message will be printed in the Webots console mentioning the missing dependency.
```

Webots also needs the *ffmpeg* and *libfdk-aac1* (from *ubuntu-restricted-extras* for H.264 codec) packages to create MPEG-4 movies.

>  The following commands should work for debian / ubuntu based distributions.

Reviewer: ```suggestion
 The following commands should work for Debian / Ubuntu based distributions:
```

```sh
sudo apt-get update
sudo apt-get install libx264-dev
sudo apt-get install libfdk-aac1
```

> Using Anaconda could cause errors when video recording, as the default conda installation of *ffmpeg* does not have x264 enabled. Execute the following command to install ffmpeg with x264.

Reviewer: ```suggestion
Using Anaconda could cause errors when recording videos, as the default conda installation of *ffmpeg* does not have *x264* enabled.
Execute the following command to install *ffmpeg* with *x264* support:
```
